Dockerfile.rhel7

```diff
@@ -1,8 +1,9 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16 AS builder
+ARG TAGS=ocp
```

Comment:
please add a comment to the dockerfile linking to a doc that lists valid values of TAGS. openshift/enhancement preferred.

And any other "standard" build-args.

Reply:
AFAIK any value other than ocp means OKD